jobs/involuntary-dissolutions/Makefile

```diff
@@ -39,7 +39,7 @@ clean-test: ## clean test files
 build-req: clean ## Upgrade requirements
 	test -f venv/bin/activate || python3.8 -m venv  $(CURRENT_ABS_DIR)/venv ;\
 	. venv/bin/activate ;\
-	pip install --upgrade pip ;\
+	pip install pip==20.3.4 ;\
```

Comment:
Upgrading pip to latest 20.x.x version seems to not break the unit tests and also installs properly for me. It does however appear to throw lots of deprecation warnings when running the unit tests. I am not sure if this is directly related to the pip version, or if it is because I am manually installing some `legal_api` dependencies that are not getting properly installed like `nest_asyncio`.

<img width="1111" alt="Screenshot 2024-06-05 at 7 47 47 AM" src="https://github.com/bcgov/lear/assets/122323255/9342b40c-c716-4ffb-aa54-e8d67de848a8">